src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms ease;
```

Comment:
set transition to define properties and move duration to variable

Reply:
It crashes when I try to create variable, only works when in exact file... So should it be like this?

Reply:
That is because you should make `main.scss` file and import all your blocks and utils there

Reply:
And if you have some questions, ask in fe_chat

Reply:
Understood. It`s just import "variables" has to be on top)